OOP part 1/code/src/Item.ts

```diff
@@ -0,0 +1,78 @@
+import { Comparable } from './Comparable';
+
+let id = 0;
+let counter = 0;
```

Comment:
If the counter represents the number of items then it should be in the Inventory class and the same for Id